src/aerie_cli/commands/goals.py

```diff
@@ -0,0 +1,129 @@
+import typer
+import os
+from typing import Optional
+
+from aerie_cli.commands.command_context import CommandContext
+
+app = typer.Typer()
+
+def _get_name_and_ext(path: str):
+    path = path.strip()
+    filename = os.path.basename(path)
+    return os.path.splitext(filename)
+
+@app.command()
+def new(
+        path: str = typer.Argument(default=...),
+        description: Optional[str] = typer.Option(
+            None, '--description', '-d', help="Description metadata"
+        ),
+        public: bool = typer.Option(False, '-pub', help="Indicates a public goal visible to all users (default false)"),
+        name: Optional[str] = typer.Option(
+            None, '--name', '-n', help="Name of the new goal (default is the file name without extension)"
+        ),
+        model_id: Optional[int] = typer.Option(
+            None, '--model', '-m', help="Mission model ID of specification to add this to"
```

Comment:
Can a mission model have more than one specification? (I didn't think so...). Would it be simpler to just say `Mission model ID to associate with the scheduling goal`? "specification" is a backend concept that most users will not understand.

Reply:
Yeah there's only one default spec per model. `Mission model ID to associate with the scheduling goal` might be misleading though because we're associating the goal with the model, not the other way around.

Reply:
But yes, its fine to not mention the spec there.